src/content/self-hosted/install/releases.mdx

```diff
@@ -5,8 +5,43 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+
+## 1.4.0
+<TODO: insert release date here once it's published>
+
+### Features
+- Update the Destroy All UI 
+- Support custom resources (requests and limits) for installer jobs
+- [External Resources](/docs/reference/manifest/#external-object-optional): Now you can use external resources in your okteto manifest. They are a list of external resources to deploy with your development environment like databases, message brokers and lambda functions.
+- Get OpenId claims from ID Token to be compatible with more providers
+- Support for "resource" and "audience" OpenId configuration
+- Pod requests won't be modified by default by okteto
+- Support "virtual services" to show endpoints in the Okteto UI
+- Follow the standards for [CRDs](/docs/reference/manifest/#crds): dedicated folder, global option in chart and keep them in helm uninstall by default
+- Support for custom [affinities](/docs/reference/manifest/#affinity) for dev containers and installer jobs in helm config
+- Add confirmation on sleep action in the UI
+- Add external endpoints to preview view
+- Add force/retry buttons and namespace logs on namespace delete-failed
+- Merge user-provided configuration snippet with the okteto configuration snippet
```

Comment:
Whatis this one about? do we have docs or a sample somewhere?

Reply:
I think it's NGINX config. @pchico83 could you confirm?

Reply:
Yes, it's the NGINX configuration snippet, done via the annotation `"nginx.ingress.kubernetes.io/configuration-snippet"`.
Before this change, we were overwriting the configuration snippet provided by the user. Now, we merge the user configuration with our configuration